themes/default/content/docs/clouds/gcp/get-started/_index.md

```diff
@@ -12,8 +12,1376 @@ menu:
+```
+
+{{% /choosable %}}
+
+{{% choosable language typescript %}}
+
+Now that `index.html` is in your bucket, modify the program file to have the bucket serve `index.html` as a static website.
+
+First, set the `website` property on your bucket. And, to align with Google Cloud Storage recommendations, set uniform bucket-level access on the bucket to `true`.
+
+```typescript
+const bucket = new gcp.storage.Bucket("my-bucket", {
+    website: {
+        mainPageSuffix: "index.html"
+    },
+    uniformBucketLevelAccess: true
+});
+```
+
+Next, allow the contents of your bucket to be viewed anonymously over the Internet.
+
+```typescript
+const bucketIAMBinding = new gcp.storage.BucketIAMBinding("my-bucket-IAMBinding", {
+    bucket: bucket.name,
+    role: "roles/storage.objectViewer",
+    members: ["allUsers"]
+});
+```
+
+Also, change the content type of your `index.html` object so that it is served as HTML.
+
+```typescript
+const bucketObject = new gcp.storage.BucketObject("index.html", {
+    bucket: bucket.name,
+    contentType: "text/html",
+    source: new pulumi.asset.FileAsset("index.html")
+});
+```
+
+Finally, at the end of the program file, export the resulting bucket’s endpoint URL so you can easily access it:
+
+```typescript
+export const bucketEndpoint = pulumi.concat("http://storage.googleapis.com/", bucket.name, "/", bucketObject.name);
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+Now that `index.html` is in your bucket, modify the program file to have the bucket serve `index.html` as a static website.
+
+First, set the `website` property on your bucket. And, to align with Google Cloud Storage recommendations, set uniform bucket-level access on the bucket to `True`.
+
+```python
+bucket = storage.Bucket('my-bucket',
+    website=storage.BucketWebsiteArgs(
+        main_page_suffix='index.html'),
+    uniform_bucket_level_access=True,
+)
+```
+
+Next, allow the contents of your bucket to be viewed anonymously over the Internet.
+
+```python
+bucketIAMBinding = storage.BucketIAMBinding('my-bucket-IAMBinding',
+    bucket=bucket.name,
+    role="roles/storage.objectViewer",
+    members=["allUsers"]
+)
+```
+
+Also, change the content type of your `index.html` object so that it is served as HTML.
+
+```python
+bucketObject = storage.BucketObject(
+    'index.html',
+    bucket=bucket.name,
+    content_type='text/html',
+    source=pulumi.FileAsset('index.html')
+)
+```
+
+Finally, at the end of the program file, export the resulting bucket’s endpoint URL so you can easily access it:
+
+```python
+pulumi.export('bucket_endpoint', pulumi.Output.concat('http://storage.googleapis.com/', bucket.id, "/", bucketObject.name))
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+Now that `index.html` is in your bucket, modify the program file to have the bucket serve `index.html` as a static website.
```

Comment:
I think you need an empty line here, as there's some formatting that's getting lost: 

![image](https://github.com/pulumi/pulumi-hugo/assets/274700/2a6ebbc1-5523-4dd3-ae52-f06ba2fec5da)

```suggestion

Now that `index.html` is in your bucket, modify the program file to have the bucket serve `index.html` as a static website.
```